Lib/Xamarin.Forms.OpenTok/Service/BaseOpenTokService.cs

```diff
@@ -14,6 +14,8 @@ public abstract class BaseOpenTokService : IOpenTokService
 
         public event Action<string> MessageReceived;
 
+        public event Action<string> SubscriberAdded;
```

Comment:
What do you think, if we remove this event and add property ObservableCollection<string> or ObservableCollection<OpenTokSubscriberInfo> ?

```public ObservableCollection<OpenTokSubscriberInfo> SubscribersInfo { get; } = new ...```